sdk/resourcehealth/arm-resourcehealth/CHANGELOG.md

```diff
@@ -1,15 +1,118 @@
+  - Added Type Alias EventGetBySubscriptionIdAndTrackingIdResponse
+  - Added Type Alias EventGetByTenantIdAndTrackingIdResponse
+  - Added Type Alias EventLevelValues
+  - Added Type Alias EventsListBySingleResourceNextResponse
+  - Added Type Alias EventsListBySingleResourceResponse
+  - Added Type Alias EventsListBySubscriptionIdNextResponse
+  - Added Type Alias EventsListBySubscriptionIdResponse
+  - Added Type Alias EventsListByTenantIdNextResponse
+  - Added Type Alias EventsListByTenantIdResponse
+  - Added Type Alias EventSourceValues
+  - Added Type Alias EventStatusValues
+  - Added Type Alias EventTypeValues
+  - Added Type Alias ImpactedResourcesGetByTenantIdResponse
+  - Added Type Alias ImpactedResourcesGetResponse
+  - Added Type Alias ImpactedResourcesListBySubscriptionIdAndEventIdNextResponse
+  - Added Type Alias ImpactedResourcesListBySubscriptionIdAndEventIdResponse
+  - Added Type Alias ImpactedResourcesListByTenantIdAndEventIdNextResponse
+  - Added Type Alias ImpactedResourcesListByTenantIdAndEventIdResponse
+  - Added Type Alias IssueNameParameter
+  - Added Type Alias LevelValues
+  - Added Type Alias LinkTypeValues
+  - Added Type Alias MetadataGetEntityResponse
+  - Added Type Alias MetadataListNextResponse
+  - Added Type Alias MetadataListResponse
+  - Added Type Alias ReasonTypeValues
+  - Added Type Alias Scenario
+  - Added Type Alias SecurityAdvisoryImpactedResourcesListBySubscriptionIdAndEventIdNextResponse
+  - Added Type Alias SecurityAdvisoryImpactedResourcesListBySubscriptionIdAndEventIdResponse
+  - Added Type Alias SecurityAdvisoryImpactedResourcesListByTenantIdAndEventIdNextResponse
+  - Added Type Alias SecurityAdvisoryImpactedResourcesListByTenantIdAndEventIdResponse
+  - Interface AvailabilityStatusProperties has a new optional parameter articleId
+  - Interface AvailabilityStatusProperties has a new optional parameter category
+  - Interface AvailabilityStatusProperties has a new optional parameter context
+  - Interface AvailabilityStatusProperties has a new optional parameter recentlyResolved
+  - Interface AvailabilityStatusProperties has a new optional parameter title
+  - Interface RecommendedAction has a new optional parameter actionUrlComment
+  - Interface Resource has a new optional parameter systemData
+  - Added Enum KnownAvailabilityStateValues
+  - Added Enum KnownCreatedByType
+  - Added Enum KnownEventLevelValues
+  - Added Enum KnownEventSourceValues
+  - Added Enum KnownEventStatusValues
+  - Added Enum KnownEventTypeValues
+  - Added Enum KnownIssueNameParameter
+  - Added Enum KnownLevelValues
+  - Added Enum KnownLinkTypeValues
+  - Added Enum KnownReasonChronicityTypes
+  - Added Enum KnownReasonTypeValues
+  - Added Enum KnownScenario
 
-### Other Changes
+**Breaking Changes**
 
+  - Operation EmergingIssues.get has a new signature
+  - Interface AvailabilityStatusProperties no longer has parameter recentlyResolvedState
```

Comment:
@kazrael2119 Could you help check the breaking from?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/22721#issuecomment-1490047589

Reply:
the first breaking:https://github.com/Azure/azure-rest-api-specs/pull/22479

Reply:
Since this breaking can't be avoided so we'll accept them